packages/core/framework/src/http/__tests__/validate-body.spec.ts

```diff
@@ -1,7 +1,7 @@
-import zod from "zod"
 import { MedusaError } from "@medusajs/utils"
-import { validateAndTransformBody } from "../utils/validate-body"
+import zod, { ZodNullable, ZodObject, ZodOptional } from "zod"
```

Comment:
question: did we change these, because they are no longer available on the `zod` object?